lib/realtime/vehicle.ex

```diff
@@ -1,15 +1,30 @@
 defmodule Realtime.Vehicle do
+  alias Gtfs.{Direction, Route, Stop, Timepoint, Trip}
+
   @type current_status() :: :in_transit_to | :stopped_at
 
+  @type stop_timepoint_status() :: %{
+          stop_granularity: %{
+            current_status: current_status(),
+            stop_id: Stop.id()
+          },
+          timepoint_granularity: %{
+            current_status: current_status(),
```

Comment:
I'm not sure that this is a totally meaningful concept for timepoints. I think all our vertical positioning could be based on the percentage (especially if we make more accurate ways to calculate the percentage).